### PR TITLE
QPD-6272: Remove storing of tokens in files and use instance vars

### PR DESCRIPTION
**Pull request by tushar-quartic**

### CODE REVIEW

**Review by scriperdjq — APPROVED** [review]

**Comment by scriperdjq** [comment]
@tushar-quartic can u address to the sourcery's comments?

File: quartic_sdk/graphql_client.py [hunk 1]
```diff
@@ -49,11 +49,10 @@ class initialisation
         self.url = url
         self.username = username
         self.password = password
-        self.token = token
         self.verify_ssl = verify_ssl
         self.timeout = timeout
         self.__graphql_url = self._get_graphql_url()
```
[the remark this answers is not in the document]
Comment (tushar-quartic) [reply quartic_sdk/graphql_client.py:55]: yes, this has been used inside utilities/decoraters.py for fetching new access tokens

File: quartic_sdk/api/api_helper.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def __init__(self, host, username=None, password=None, oauth_token=None, cert_pa
         """
         self.configuration = Configuration.get_configuration(
             host, username, password, oauth_token, cert_path, verify_ssl, gql_host)
-        self.access_token = get_and_save_token(host,username,password,verify_ssl)
```
[the remark this answers is not in the document]
Comment (tushar-quartic) [reply quartic_sdk/api/api_helper.py:20]: yes, this has been used inside utilities/decoraters.py for fetching new access tokens

File: quartic_sdk/utilities/decorator.py [hunk 3]
```diff
@@ -91,10 +80,9 @@ def request_new_token(refresh_token, host,user_identification_string):
     Raises:
         PermissionError: If the refresh token has expired or if any other error occurs during the token request.
     """
-    try:
```
[the remark this answers is not in the document]
Comment (tushar-quartic) [reply quartic_sdk/utilities/decorator.py:94]: Already added try except block to raise any unnecessary errors. The errors regarding token handling will throw only 400 and 401 status codes.

**Comment by tushar-quartic** [comment]
> @tushar-quartic can u address to the sourcery's comments?

Done

**Review by manoj016 — APPROVED** [review]

**Pull request merged**